https/docker-compose.yml

```diff
@@ -0,0 +1,170 @@
+      - "traefik.port=8989"
+      - "traefik.enable=true"
+      - "traefik.https.frontend.rule=Host:sonarr.${DOMAIN}" # delete for no remote access
+      - "traefik.https.frontend.entryPoints=https" # delete for no remote access
+      - "traefik.https.frontend.auth.basic.users=${USERNAME}:${PASSWORD}" # delete for no remote access
+
+  radarr:
+    container_name: radarr
+    image: linuxserver/radarr:latest
+    restart: unless-stopped
+    expose:
+      - "7878"
+    environment:
+      - PUID=${PUID} # default user id, defined in .env
+      - PGID=${PGID} # default group id, defined in .env
+      - TZ=${TZ} # timezone, defined in .env
+    volumes:
+      - /etc/localtime:/etc/localtime:ro
+      - ${ROOT}/config/radarr:/config # config files
+      - ${ROOT}/complete/movies:/movies # movies folder
+      - ${ROOT}/downloads:/downloads # download folder
+    labels:
+      - "traefik.backend=radarr"
+      - "traefik.local.frontend.rule=Host:radarr.localhost"
+      - "traefik.port=7878"
+      - "traefik.enable=true"
+      - "traefik.https.frontend.rule=Host:radarr.${DOMAIN}" # delete for no remote access
+      - "traefik.https.frontend.entryPoints=https" # delete for no remote access
+      - "traefik.https.frontend.auth.basic.users=${USERNAME}:${PASSWORD}" # delete for no remote access
+
+  plex-server:
+    container_name: plex-server
+    image: plexinc/pms-docker:latest
+    restart: unless-stopped
+    environment:
+      - TZ=${TZ} # timezone, defined in .env
+    network_mode: host
+    volumes:
+      - ${ROOT}/config/plex/db:/config # plex database
+      - ${ROOT}/config/plex/transcode:/transcode # temp transcoded files
+      - ${ROOT}/complete:/data # media library
+
+  traefik:
+    container_name: traefik
+    image: traefik:latest
+    restart: always
+    command: --logLevel=info --entryPoints="Name:http Address::80" --defaultEntryPoints=https,http \
+             --docker --docker.domain=${DOMAIN} --docker.endpoint="unix:///var/run/docker.sock" \
+             --docker.watch=true --docker.exposedbydefault=false \
+             --api \
+             --entryPoints="Name:https Address::443 TLS" \
+             --acme=true --acme.entrypoint=https --acme.httpchallenge --acme.httpchallenge.entrypoint=http \
+             --acme.domains=${DOMAIN} --acme.email=${EMAIL} --acme.storage=acme.json
+    ports:
+      - 80:80
+      - 443:443
+      - 8080:8080
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+      - ${ROOT}/config/traefik/acme.json:/acme.json
+    labels:
+      - "traefik.backend=traefik"
+      - "traefik.local.frontend.rule=Host:traefik.localhost"
+      - "traefik.port=8080"
+      - "traefik.enable=true"
+      - "traefik.https.frontend.rule=Host:traefik.${DOMAIN}" # delete for no remote access
+      - "traefik.https.frontend.entryPoint=https" # delete for no remote access
+      - "traefik.https.frontend.auth.basic.users=${USERNAME}:${PASSWORD}" # delete for no remote access
```

Comment:
It's too bad we duplicate most of the file in both `https` and `http` (now we need to fix both everytime).
What do you think about keeping a single file, but having the https lines commented?

Reply:
I considered that, and it is a strong motivation (took a while working out the traefik config so we didn't have to use a traefik.toml file). However, there are some differences that are a bit less trivial, (like editing the traefik command configuration section), and since I've been working on Organizr support, there is a bit of configuration difference when adding that too.

I also like that there won't be any clutter for people using the `base` version in the docker-compose file or the .env file.

More importantly, requiring editing to add https make upgrading from further updates more difficult, and being able to update just by `git pull`ing is something that seems pretty desirable.

Since the two files are interoperable, in the sense that they don't conflict when running from the same ROOT directory, I think it is OK to keep both files. I don't find it that big of a deal to update each one.

Reply:
Yeah sounds good to me, we can keep both.